packages/utilities/psammead-storybook-helpers/src/input-provider.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Fragment } from 'react';
+import { text, select } from '@storybook/addon-knobs';
+import { Helmet } from 'react-helmet';
+import scripts from '@bbc/gel-foundations/scripts';
+import LANGUAGE_VARIANTS from './text-variants';
+
+const inputProvider = (slots, componentFunction) => () => {
+  const lang = select(
+    'Select a language',
+    LANGUAGE_VARIANTS,
+    LANGUAGE_VARIANTS.English,
+  );
+
+  // `select` doesn't return name of language selected, so test if selection is English
+  // by comparing `text` to English's `text`
+  const isEnglish = lang.text === LANGUAGE_VARIANTS.English.text;
+
+  const inputs = (slots || []).map(({ name, defaultText }) =>
+    text(
+      `Content for ${name}`,
+      // Only display slot default text when it exists and we're displaying English language.
```

Comment:
Could you elaborate a bit more when the `isEnglish` check is needed?

Reply:
I've added some more to this comment, does that help?